Lib/test/test_doctest.py

```diff
@@ -3039,6 +3039,8 @@ def test_CLI(): r"""
     ...         '-m', 'doctest', 'nosuchfile')
     >>> rc, out
     (1, b'')
+    >>> if sys.platform == "vxworks":
+    ...     err = err.replace(b'no such file', b'No such file')
```

Comment:
bummer that https://bugs.python.org/issue13337 hasn't been implemented.  rather than relying on the error message text at all though, why not just use more ... below.  The FileNotFoundError already indicates the error message and is canonical.  i suspect this was written before python 3 added that exception heirarchy.

would `FileNotFoundError: [Errno ...] ... nosuchfile...` work?

Reply:
`FileNotFoundError: [Errno ...] ...nosuchfile...` can work. Remove the extra space just before nosuchfile. Thanks for better solution.